src/pages/designing/kits/figma.mdx

```diff
@@ -52,33 +51,33 @@ the toolbar (A). Open the menu and select **Libraries** from the list (B).
 <Row>
 <Column colLg={8}>
 
-![internal figma instructions part 1](images/kit-figma-1.png)
+![internal figma instructions part 1](images/design-figma-1.png)
 
 </Column>
 </Row>
 
 Then in the Libraries modal that appears, find the team called
 `IBM Design Systems` (C) and switch the toggle beside the
-`(v11) All Themes - Carbon Design System` library to on (D).
+`(v11) Carbon Design System` library to on (D).
 
 <Row>
 <Column colLg={8}>
 
-![internal figma instructions part 2](images/kit-figma-2.png)
+![internal figma instructions part 2](images/design-figma-2.png)
 
 </Column>
 </Row>
 
 <br />
 
-To preview the All themes library, visit the following view-only link. To access
-the v10 Figma theme libraries, see
+To preview the Carbon Design System library, visit the following view-only link.
+To access the v10 Figma theme libraries, see
```

Comment:
I know this isn't directly related but I think since we just reached end-of-support for v10 we can actually just remove this whole v10 sentence.